src/Store.js

```diff
@@ -519,6 +525,11 @@ class Store {
   _addOperationBatch (data, batchOperation, lastOperation, onProgressCallback) {
     throw new Error('Not implemented!')
   }
+  
+  _procEntry(entry) {
+    var { op, value } = entry.payload;
```

Comment:
As of right now `orbit-db-store` doesn't strictly enforce the `{ op, value }` schema. If you can add some validation and make this a noop if `op` isn't defined, I think we'd be good to go.

Reply:
@aphelionz Yes definitely! That does raise the question what about operations in regards to the access controller?

Reply:
Good question - what are your initial thoughts there?

Reply:
It appears the currently existing set of access controllers use a separate database for AC related activities. It shouldn't conflict this.